doc/source/installation.rst

```diff
@@ -107,6 +107,18 @@ Then install the Python packages in the requirements.txt file
     sudo pip3 install -r requirements.txt
     sudo pip3 install -r src/neem_interface_python/requirements.txt
 
+This installs the packages into ``/usr/local/lib``. If you prefer to not clutter your system-wide python installation,
+you can also install the packages into the catkin workspace as follows:
+
+.. code-block:: shell
+
+    # install packages into catkin workspace instead of ~/.local
+    export PYTHONUSERBASE=~/workspace/ros/devel
+    # don't install packages that are available in system
+    export PIP_IGNORE_INSTALLED=0
+
+    pip3 install -r requirements.txt
+    pip3 install -r src/neem_interface_python/requirements.txt
```

Comment:
Please remove this line, since the neem_interface_python is not part of the repo anymore.

Reply:
Ok. I didn't use that and thus didn't notice.